discovery.go

```diff
@@ -190,14 +194,18 @@ func (d *discover) Bootstrap(ctx context.Context, topic string, ready RouterRead
 		return true
 	}
 
+	t := time.NewTimer(time.Hour)
+	if !t.Stop() {
```

Comment:
This is the most pessimistic check I've ever seen. I like it :laughing:.

Reply:
wait what? this seems completely unnecessary.

Reply:
It is likely unnecessary given how long the timer is set for. However, it is basically 0 performance cost for properly using the `Timer.Stop()` idiom as is described in the documentation.